resources/user-guides/index.md

```diff
@@ -67,7 +67,7 @@ title: "User Guides"
                 </div>
 <div class="row start-xs text-center" markdown="1">
 
-User Guides for Mining coming soon.
+[How to mine on a pool](mine-to-pool.html)
```

Comment:
I would change it in ```How to mine on a pool (with xmr-stak-cpu)```, because the guide is very specific and cannot be used for another miner